wavefront_python_sdk/entities/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import
+
+# flake8: noqa
+
+from wavefront_python_sdk.entities.histogram import histogram_granularity
```

Comment:
Is there any reason to keep importing this into `__init__.py` file?

Reply:
So that It can be directly `from wavefront_python_sdk.entities import histogram_granularity`

Reply:
I see, so you want it to be exposed as `wavefront_python_sdk.entities.histogram_granularity` instead of `wavefront_python_sdk.entities.histogram.histogram_granularity`

Reply:
Yes, it feels like we don't necessarily need one extra `.histogram.`